src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -112,6 +120,24 @@ export default function createPostGraphQLHttpRequestHandler (options) {
       return
     }
 
+    // Serve the JavaScript for GraphiQL on a namespaced path
+    if (parseUrl(req).pathname === '/_postgraphql/graphiql.js') {
+      res.statusCode = 200
+      res.setHeader('Content-Type', 'text/javascript')
+      res.end(await graphiqlJs)
+      return
+    }
+
+    // Setup an event stream so we can broadcast events to graphiql, etc.
+    if (parseUrl(req).pathname === '/_postgraphql/stream') {
+      if (req.headers.accept !== 'text/event-stream') {
+        res.end()
+        return
+      }
+      setupServerSentEvents(req, res, options)
```

Comment:
@calebmer I wasn’t entirely sure how to best extract that out because of the control flow stuff that is going on in the request handler. Is there a better way?